src/plugins/taxes-rates/resolvers/Mutation/createTaxRate.js

```diff
@@ -0,0 +1,34 @@
+import { decodeShopOpaqueId } from "../../xforms/id.js";
```

Comment:
@aldeed what is the recommended pattern to decode opaque ids? importing utility functions in this way, seems to couple plugins with core. How about importing from `import decodeOpaqueIdForNamespace from "@reactioncommerce/api-utils/decodeOpaqueIdForNamespace.js";` and using ` const shopId = decodeOpaqueIdForNamespace("reaction/shop", opaqueShopId);` ?

Reply:
@willopez This imports from its own package. The id.js file imports from "api-utils". This is the same pattern used in the other plugins.